enhancements/monitoring/alerting-consistency.md

```diff
@@ -0,0 +1,195 @@
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Alerts are a mess.  There's not really formal guidelines to follow when
+implementing alerts for a given component.  These are the guidelines.
+
+## Motivation
+
+Improve Signal to noise of alerts.  Align alert levels "Critical, Warning, Info"
+with operational expectations to allow easier triaging.  Ensure current and
+future alerts have documentation covering problem/investigation/solution steps.
+
+### Goals
+
+* Define clear criteria for designating an alert 'critical'
+* Define clear criteria for designating an alert 'warning'
+* Define minimum time thresholds before prior to triggering an alert
+* Establish clear documentation guidelines
+* Outline operational triage of firing alerts
+
+### Non-Goals
+
+* Define specific alert needs
+* Implement user-defined alerts
+
+## Proposal
+
+### User Stories
+
+#### Story 1
+
+As an OpenShift developer, I need to ensure that my alerts are appropriately
+tuned to allow end-user operational efficiency.
+
+#### Story 2
+
+As an OpenShift administrator, I need to have confidence that alerts are
+actually informative and actionalble to maintain my clusters.
+
+### Implementation Details/Notes/Constraints [optional]
+
+This enhancement will require some developers and administrators to rethink
+their concept of alerts and alert levels.  Many people seem to have an
+intuition about how alerts should work that doesn't necessarily match how
+alerts actually work or how others believe alerts should work.
+
+Formalizing this guidance into a type of template or specification will allow
+developers and administrators to speak a common language and have a common
+understanding, even if the concepts seem initially unituitive.
+
+### Risks and Mitigations
+
+People will make wrong assumptions about alerts and how to handle them.  This
+is not unique to alerts, people often believe certain components should behave
+in a particular way in which they do not.  This is an artifact of poor
+documentation that leaves too much to the imagination.
+
+We can work around this problem with clear and concise documentation.
+
+## Design Details
+
+### Critical Alerts
+
+TL/DR:  Fix this now or cluster dies.
+
+Timeline:  ~5 minutes.
+
+Critical Alerts should be reserved for situations that jeopardize the health
+if not immediately addressed or if the cluster is already in a critical state.
+
+Some critical states are:
+* loss or impending loss of etcd-quorum.
+* inability to route application traffic externally or internally
+(dataplane disruption)
+* inability to start any new pod other than capacity.  EG, if the API server
+were to restart or need to be rescheduled, it would not able to start on another
+host
+
+In otherwords, critical alerts are something that require someone to get out
+of bed in the middle of the night and fix something **right now** or they will
+be faced with a disaster.
```

Comment:
clarify "disaster"

Reply:
Something that requires use of disaster recovery docs.

Reply:
Clarify in the proposal if possible please